tsconfig.json

```diff
@@ -6,7 +6,6 @@
     "moduleResolution": "node",
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
-    "strictPropertyInitialization": false,
```

Comment:
Should we also remove it from the `tsconfig-build` config files?

Reply:
Didn't realize those existed 😳

Reply:
I don't think we actually use those tsconfig-build.json files anymore.